eval/evaluation.py

```diff
@@ -0,0 +1,330 @@
+import sys
+import os
+import json
+import time
+from copy import deepcopy
+from collections import defaultdict
+
+from run_bot import arg_parse
+from moviebot.agent.agent import Agent
+from moviebot.utterance.utterance import UserUtterance
+
+
+def get_data_path(filename):
+    return os.path.join(os.getcwd(), 'eval', 'data', filename)
+
+
+def load_json(filename):
+    """Load contents of a json file.
```

Comment:
Everywhere in this file: The docstring should be descriptive-style ("Loads the contents...") rather than imperative-style ("Load contents..."), cf. https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods